lms/djangoapps/verify_student/tests/test_integration.py

```diff
@@ -42,7 +42,7 @@ def setUp(self):
             'verify_student_start_flow': reverse(
                 'verify_student_start_flow',
                 args=[unicode(self.course_key)]
-            ),
+            ) + purchase_workflow,
```

Comment:
I had the string originally but felt the variable helped to make things easier to read.